doc/config.md

```diff
@@ -0,0 +1,97 @@
+
+The transitions parameters are all optional parameters, used for running scenario projections.
+
+##### `:filter-transitions-from`
+
+Expects a calendar year stored in a vector (e.g. `[2016]`). Sets a year to filter historic transitions from (e.g. for `[2016]` all transitions data prior to 2016 will be ignored. To be used with with `:splice-ncy` for a [“Ignore historic data before a specific calendar year for an age group”](https://github.com/MastodonC/witan.send/blob/master/doc/scenarios.md#ignore-historic-data-before-a-specific-calendar-year-for-an-age-group) scenario.
+
+##### `:splice-ncy`
+
+NCY = National Curricullum Year. 
+
+Expects an integer corresponding to the national curricullum year/academic year to filter from (typically between -5 and 20). To be used in conjunction with `:filter-transitions-from` for a [“Ignore historic data before a specific calendar year for an age group”](https://github.com/MastodonC/witan.send/blob/master/doc/scenarios.md#ignore-historic-data-before-a-specific-calendar-year-for-an-age-group) scenario. Currently will filter the provided NCY and all years above (e.g. for 16, NCYs 16-20 will be filtered).
+
+##### `:which-transitions?`
+
+Expects a vector (`["foo"]`) of either one or more transition types stored as strings i.e. "joiners", "movers-to", "movers-from" or "leavers", to be used in conjunction with `:modify-transition-by` and `:setting-to-change` parameter with which to define a [“Modify setting(s) transitions rates”](https://github.com/MastodonC/witan.send/blob/master/doc/scenarios.md#modify-settings-transitions-rates), [“Modify setting(s) transitions rates and transfer individuals to alternative setting(s)”](https://github.com/MastodonC/witan.send/blob/master/doc/scenarios.md#modify-settings-transitions-rates-and-transfer-individuals-to-alternative-settings) or [“Modify transitions from a specific future calendar year”](https://github.com/MastodonC/witan.send/blob/master/doc/scenarios.md#modify-transitions-from-a-specific-future-calendar-year) scenario.
+
+##### `:modify-transition-by`
+
+Expects an integer to multiply a list of settings by (provided with `:settings-to-change`). For example here a `2` will multiply the transition rate for a list of settings by two, thus doubling the historic number of transitions by.
+
+### `:run-parameters`
+
+Contains three required and one optional key.
+
+The three required keys, `:random-seed`, `:simulations` and `:seed-year` define how the model will be run. `:modify-transitions-from` is optional and used in conjunction with `:transitions-parameters`.
+
+##### `:random-seed`
+
+Expects and integer with which to randomly seed the model with and enable repeatability.
+
+##### `:simulations`
+
+Expects an integer corresponding to a number with which to repeat the model over _n_ number of times. Due to parallelisation, the minimum number of simulations is eight. In tests 1000 simulations typically provides sufficient depth to normalise the results.
```

Comment:
"Due to how we currently do parallelisation..." The minimum of 8 is a weird thing in how we do our code.

Reply:
Yes it is. I didn't want to change any process or code yet, just document how stuff is currently done.